packages/py-moose-lib/moose_lib/dmv2/olap_dictionary.py

```
@@ -0,0 +1,334 @@
+            primary_key=["user_id"],
+            layout={"type": "HASHED"},
+            lifetime=3600,
+        )
+
+        # Use in a query
+        sql = f"dictGet('{user_dict.name}', 'display_name', user_id)"
+    """
+
+    kind: str = "OlapDictionary"
+
+    def __init__(
```

Ruff (0.15.7)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)